el9.tpl

```diff
@@ -0,0 +1,214 @@
+config_opts['root'] = 'terra-{{ releasever }}-{{ target_arch }}'
```

Comment:
```suggestion
config_opts['root'] = 'terra-el{{ releasever }}-{{ target_arch }}'
```
that way we can distinguish it from Fedora Terra

Reply:
removed el there originally because i thought the releasever needed it, forgot to readd that